### PR TITLE
add sklearn2pmml

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by FernandezMathieu** [comment]
Requirements `sklearn-pandas` [in a PR.](https://github.com/conda-forge/staged-recipes/pull/13335)

**Comment by FernandezMathieu** [comment]
@conda-forge-linter

**Comment by FernandezMathieu** [comment]
@conda-forge-linter restart ci

**Comment by FernandezMathieu** [comment]
@conda-forge-linter restart ci

**Comment by FernandezMathieu** [comment]
@conda-forge-linter restart ci

File: recipes/sklearn2pmml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - sklearn2pmml.decoration
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jpmml/sklearn2pmml
+  summary: Python library for converting Scikit-Learn pipelines to PMML
+  license: GPL-3.0
```
Comment (dopplershift) [inline recipes/sklearn2pmml/meta.yaml:40]: The license looks like it's actually AGPL-3.0

File: recipes/sklearn2pmml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - joblib >=0.13.0
+    - python
```
Comment (dopplershift) [inline recipes/sklearn2pmml/meta.yaml:24]: Should also be `python >=2.7`

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor things.

File: recipes/sklearn2pmml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - sklearn2pmml.decoration
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jpmml/sklearn2pmml
+  summary: Python library for converting Scikit-Learn pipelines to PMML
+  license: AGPL-3.0
```
Comment (dopplershift) [inline recipes/sklearn2pmml/meta.yaml:40]: Apologies, looks like that actually should have been `AGPL-3.0-only`

**Review by dopplershift — APPROVED** [review]

**Pull request merged**